### PR TITLE
DDF-2141: Update Example Query Response table and Activity Events table

### PR DESCRIPTION
**Pull request by Lambeaux**
#### What does this PR do?
Updates the integration docs with corrections and format fixes. 

#### Who is reviewing it?
@ricklarsen 
@oconnormi 
@jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic
@shaundmorris

#### How should this be tested?
Read it. 
Okay, maybe build DDF docs and ensure HTML/PDF renders fine. 

#### Any background context you want to provide?
No. 

#### What are the relevant tickets?
DDF-2141

#### Checklist:
- [x] Documentation Updated

### CODE REVIEW

**Comment by shaundmorris** [comment]
@Lambeaux please add a title to the PR

File: distribution/docs/src/main/resources/_contents/integrating-contents.adoc [hunk 1]
```diff
@@ -879,76 +878,62 @@ The response is returned as a data map that contains an internal map with the fo
...
+                        "created": "2016-06-09T22:46:37.707+0000",
+                        "metacard-tags": [
+                            "resource"
+                        ],
+                        "resource-uri": "content:1a83781dd56a4050afc420016d078e22",
+                        "checksum-algorithm": "Adler32",
+                        "metadata-content-type": "image/jpeg",
+                        "metacard-type": "ddf.metacard",
+                        "title": "fixTheBugs_geotagged.jpg",
+                        "resource-download-url": "https://localhost:8993/services/catalog/sources/ddf.distribution/1a83781dd56a4050afc420016d078e22?transform=resource",
```
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: Not sure if it is possible but wondering if you could use the new secure url property there in the resource-download-url.  @ricklarsen is that something that can be done there?

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: I think it could, once that change goes through.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: Once that PR makes it in

**Comment by ricklarsen** [comment]
👍

**Comment by shaundmorris** [comment]
Thinking we make more updates to the table while we are there? We have had a few new fields in the query response. Should we add metacard-types, metacard-tags into the table?

**Comment by Lambeaux** [comment]
I'm on board with that. I'll make some notes on the PR - there is still some smoke we need to clear.

File: distribution/docs/src/main/resources/_contents/integrating-contents.adoc (hunk 1 above)
Comment (Lambeaux) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: Probably not a good idea. This is an example pulled right from the CometD endpoint. We want integrators to take it literally. Do we want a property substitution in the URL if you'd normally never see that?

**Comment by shaundmorris** [comment]
@Lambeaux maybe we just need to think about what fields integrators would be the most interested in. I don't think every field needs to document in verbose detail.

**Comment by Lambeaux** [comment]
What is `event.topics` used for at line 530? It doesn't have any values in its table row.

**Comment by Lambeaux** [comment]
Fields have been added to provide more comprehensive coverage of the JSON response.

File: distribution/docs/src/main/resources/_contents/integrating-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: The property substitution will happen during rendering of the docs. It'll resolve to `https://localhost:8993` in the final version. It's like the `${branding}` property. See this PR: https://github.com/codice/ddf/pull/845

**Comment by tbatie** [comment]
:+1: hero'd successfully

**Comment by Lambeaux** [comment]
@lessarderic ready for merge on your mark

**Comment by Lambeaux** [comment]
🎉

**Pull request merged**